components/server/src/server.rs

```diff
@@ -243,6 +247,9 @@ impl<ER: RaftEngine> TiKVServer<ER> {
         let latest_ts = block_on(pd_client.get_tso()).expect("failed to get timestamp from PD");
         let concurrency_manager = ConcurrencyManager::new(latest_ts);
 
+        let disk_reserved = config.storage.reserve_space.0;
+        disk::set_disk_reserved(disk_reserved);
```

Comment:
The `config.storage.reserve_space` can't be changed online, maybe it is better to pass the config to the worker instead of setting & reading from the atomic value.

Reply:
we have resereved a replica value, to be used by the worker.
and after this pr, we will set up another pr to deal with the reserve_space and the place holder file.

Reply:
We don't need to `set_disk_reserved` and `get_disk_reserved` since the `reserve_space` not support changing online, we can just pass the config value to the worker and use it directly.

Reply:
> we can just pass the config value to the worker and use it directly

`disk_reserved` is also used in `fn disk_is_full`. If it's not stored in a global available place, we need to pass a parameter into `disk_is_full(reserved)`.

Reply:
good advise, I'll simplify it.